plugins/catalog-backend/src/database/operations/stitcher/performStitching.ts

```diff
@@ -212,11 +213,12 @@ export async function performStitching(options: {
       final_entity: JSON.stringify(entity),
       hash,
       last_updated_at: knex.fn.now(),
+      entity_ref: entityRef,
     })
     .where('entity_id', entityId)
     .where('stitch_ticket', stitchTicket)
     .onConflict('entity_id')
-    .merge(['final_entity', 'hash', 'last_updated_at']);
+    .merge(['final_entity', 'hash', 'last_updated_at', 'entity_ref']);
```

Comment:
this SHOULD be impossible to change after the fact - could probably revert this line